cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -10410,3 +10410,73 @@ struct_init.cpp:
 #   36|     v36_9(void)           = ReturnVoid                       : 
 #   36|     v36_10(void)          = AliasedUse                       : ~m?
 #   36|     v36_11(void)          = ExitFunction                     : 
+
+try_except.c:
+#    6| void f()
+#    6|   Block 0
+#    6|     v6_1(void)            = EnterFunction                  : 
+#    6|     mu6_2(unknown)        = AliasedDefinition              : 
+#    6|     mu6_3(unknown)        = InitializeNonLocal             : 
+#    7|     r7_1(glval<int>)      = VariableAddress[x]             : 
+#    7|     mu7_2(int)            = Uninitialized[x]               : &:r7_1
+#    7|     r7_3(glval<int>)      = VariableAddress[y]             : 
+#    7|     r7_4(int)             = Constant[0]                    : 
+#    7|     mu7_5(int)            = Store[y]                       : &:r7_3, r7_4
+#    9|     r9_1(glval<unknown>)  = FunctionAddress[ProbeFunction] : 
+#    9|     r9_2(int)             = Constant[0]                    : 
+#    9|     v9_3(void)            = Call[ProbeFunction]            : func:r9_1, 0:r9_2
+#    9|     mu9_4(unknown)        = ^CallSideEffect                : ~m?
+#   10|     r10_1(glval<int>)     = VariableAddress[y]             : 
+#   10|     r10_2(int)            = Load[y]                        : &:r10_1, ~m?
+#   10|     r10_3(glval<int>)     = VariableAddress[x]             : 
+#   10|     mu10_4(int)           = Store[x]                       : &:r10_3, r10_2
+#   11|     r11_1(glval<unknown>) = FunctionAddress[ProbeFunction] : 
+#   11|     r11_2(int)            = Constant[0]                    : 
+#   11|     v11_3(void)           = Call[ProbeFunction]            : func:r11_1, 0:r11_2
+#   11|     mu11_4(unknown)       = ^CallSideEffect                : ~m?
+#   16|     v16_1(void)           = NoOp                           : 
+#    6|     v6_4(void)            = ReturnVoid                     : 
+#    6|     v6_5(void)            = AliasedUse                     : ~m?
+#    6|     v6_6(void)            = ExitFunction                   : 
+
+#   13|   Block 1
+#   13|     r13_1(int) = Constant[0] : 
```

Comment:
What do we do in the case of a normal C++ catch? Is that just as messy as this right now?

Reply:
Hopefully this will be fixed automagically once I fix the catch handling you helped me realize was bugged in your other comment.

Reply:
This is (partly) fixed in ff6e8a285d79b1f47d1f1c265cba28ae94741b7f. Just like for normal C++ exceptions, the `catch` block isn't actually reached by any `goto`s (this is something we really should fix in the IR eventually).

This `Constant[0]` was generated by the exception conditional in `__except(0) { ... }`, but it didn't continue to the exception block (which was definitely a bug). In `ff6e8a285d79b1f47d1f1c265cba28ae94741b7f` we now correctly model the behavior specified by https://learn.microsoft.com/en-us/cpp/cpp/try-except-statement?view=msvc-170.